pkgs/by-name/je/jellyfin-ffmpeg/package.nix

```
@@ -8,14 +8,79 @@ let
   version = "7.0.2-5";
 in
 
-(ffmpeg_7-full.override {
+(ffmpeg.override {
   inherit version; # Important! This sets the ABI.
   source = fetchFromGitHub {
     owner = "jellyfin";
     repo = "jellyfin-ffmpeg";
     rev = "v${version}";
     hash = "sha256-cqyXQNx65eLEumOoSCucNpAqShMhiPqzsKc/GjKKQOA=";
   };
+
+  ffmpegVariant = "barebones";
+
+  # https://github.com/NixOS/nixpkgs/pull/354936#issuecomment-2466628369
+  # https://github.com/jellyfin/jellyfin-ffmpeg/blob/jellyfin/debian/rules
+  # --enable-gmp # not supported by our ffmpeg drv
+  withDoc = false;
+  buildFfplay = false;
+  withXcb = false;
+  withSdl2 = false;
+  withXlib = false;
+  withGPL = true;
+  withVersion3 = true;
+  withShared = true;
```

Looking at the barebones config, those were all already disabled in it, and Shared handled by the hostPlatform

```suggestion
  withGPL = true;
  withVersion3 = true;
```

---

Same as https://github.com/NixOS/nixpkgs/pull/354936#discussion_r1837378209